kafka_helper/__init__.py

```diff
@@ -65,14 +66,18 @@ def get_kafka_ssl_context() -> ssl.SSLContext:
         key_file.write(pem)
         key_file.flush()
 
-        trust_file.write(os.environ['KAFKA_TRUSTED_CERT'].encode('utf-8'))
-        trust_file.flush()
+        if 'KAFKA_TRUSTED_CERT' in os.environ:
+            trust_file.write(os.environ['KAFKA_TRUSTED_CERT'].encode('utf-8'))
+            trust_file.flush()
+            trust_file_path = trust_file.name
+        else:
+            trust_file_path = certifi.where()
```

Comment:
Does this mean we expect the certificates to already exist (e.g. `/etc/ssl/certs/ca-certificates.crt`)? I am not familiar with SSL, so I am not sure how the certificates get installed. Otherwise, looks good to me.

Reply:
This gives the path to the cert bundle packaged with certifi itself (and so gets installed when the Python package is installed).